### DISCUSSION TITLE
Dynamic Oscilatory Shear solver

### DISCUSSION THREAD

**Discussion by thiagostellin** [fppimenta/rheoTool#50] [Q&A]
Hello guys. I have been reading the user guide but it was not clear to me: is there any solver for Dynamic Oscilatory Shear cases?
Thanks in advance.

**Comment by fppimenta** [comment]
Depending on the purpose, either rheoFoam (just setup the case) or rheoTestFoam (needs simple changes in the code; in this case, it might be easier to just integrate the equations somewhere else).
